get_session_request.php

```diff
@@ -0,0 +1,64 @@
+<?php
+require_once 'vendor/autoload.php';
+require_once 'config.php';
+
+use \Firebase\JWT\JWT;
+
+function get_session_request($contents)
+{
+    $parsedjson = json_decode($contents, true);
+    error_log($contents);
+
+    if (JWT_ENABLED) {
+        $jwt_pk = file_get_contents(ROOT_DIR . IRMA_SERVER_PUBLICKEY);
+        try {
+            $decoded = JWT::decode($contents, $jwt_pk, array('RS256'));
+
+            $disclosed = (array) $decoded->disclosed;
+            foreach ($disclosed as $con) {
+                foreach ($con as $attr) {
+                    if ($attr->id == ISSUER . ".gemeente.personalData.fullname" || $attr->id == ISSUER . '.pbdf.linkedin.familyname') {
+                        $fullname = $attr->rawvalue;
```

Comment:
This retrieves the value of the disclosed attribute in a completely different fashion than how it is retrieved when `JWT_ENABLED` is false on line 35, for no good reason that I can see. Instead of this foreach-foreach-if, you should be able to do just `$fullname = $decoded->disclosed[0][0]->rawvalue`.